client/blocks/eligibility-warnings/hold-list.tsx

```diff
@@ -288,4 +320,15 @@ function isHardBlockingHoldType(
 export const hasBlockingHold = ( holds: string[] ) =>
 	holds.some( hold => isHardBlockingHoldType( hold, getBlockingMessages( identity ) ) );
 
-export default localize( HoldList );
+export default connect(
+	( state: object ) => {
+		const siteId = getSelectedSiteId( state );
+		return {
+			siteId,
+			siteIsLaunching: getRequest( state, launchSite( siteId, null ) )?.isLoading ?? false,
```

Comment:
I think the 2nd null param is not expected by `launchSite`.
```suggestion
			siteIsLaunching: getRequest( state, launchSite( siteId ) )?.isLoading ?? false,
```

Reply:
Thanks! I think this was a copy pasta